cmd/wksctl/profile/disable/disable.go

```diff
@@ -0,0 +1,100 @@
+package disable
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/wksctl/cmd/wksctl/profile/constants"
+	"github.com/weaveworks/wksctl/pkg/git"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "disable",
+	Short: "disable profile",
+	Args:  profileDisableArgs,
+	Run:   profileDisableRun,
+}
+
+var profileDisableOptions struct {
+	repository string
+	noCommit   bool
+}
+
+func init() {
+	Cmd.Flags().StringVarP(&profileDisableOptions.repository, "repository", "", "", "enable profile from the repository")
+	Cmd.Flags().BoolVarP(&profileDisableOptions.noCommit, "no-commit", "", false, "no auto commit and push behaviour")
+}
+
+func profileDisableArgs(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
+		return errors.New("profile disable does not require any argument")
+	}
+	return nil
+}
+
+func profileDisableRun(cmd *cobra.Command, args []string) {
+	repoUrl, err := cmd.Flags().GetString("repository")
+	if err != nil {
+		log.Fatal(err)
+	}
+	if repoUrl == "" {
+		log.Fatal(errors.New("profile repository must be specified"))
+	}
+
+	if repoUrl == constants.AppDevAlias {
```

Comment:
could we make `AppDevRepoURL` the default instead of providing an alias? Also, the usual way we call this flag is `git-url`.

Reply:
fixed

Reply:
It seems that only the latter change has been made (flag rename). What I meant by _make `AppDevRepoURL` the default instead of providing an alias_ is something like this:

```go
Cmd.Flags().StringVar(&profileDisableParams.gitUrl, "git-url", constants.AppDevRepoURL, "enable profile from the Git URL")
```

and removing the following block:
```go
if repoUrl == constants.AppDevAlias {
	repoUrl = constants.AppDevRepoURL
}
```

suggestion: make the change as above

Also applies to `profile enable`.

Reply:
Dropped because @chanwit says that `app-dev` shouldn't be the default idea